string command_topic
string joint_state_topic
float32 grasped_current
---
bool success
